src/common/users.ts

```diff
@@ -327,7 +329,7 @@ export const getUserReadingRank = async (
   const now = new Date();
   const getReadingTags = () => {
     if (version === 1) {
-      return Promise.resolve(null);
+      return Promise.resolve([]);
```

Comment:
This is why snapshots were updated, the type even expected `tags` to always be array so I think more logical. No real impact on frontend.